e2e-tests/users/run

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+create_namespace $namespace
+deploy_operator
+cluster="some-name"
+kubectl_bin apply -f $conf_dir/client.yml
+
+spinup_pgcluster "${cluster}" "${src_dir}/deploy/cr.yaml"
+compare_psql_cmd "show-1" 'SHOW DATABASES' "pgbouncer:pgbouncer_pass@${cluster}-pgbouncer.${namespace}"
+compare_psql_cmd "select-1" '\c myapp \\\ SELECT * from myApp;' "postgres:postgres_pass@${cluster}.${namespace}"
+
+newpass="test-password"
+newpassencrypted=$(echo -n "$newpass" | base64)
```

Comment:
please add here some regular `some-name` user without root privileges

Reply:
added in https://github.com/percona/percona-postgresql-operator/commit/0ad14b7f1c2cc837914914116dc3ec177514d8a8